server/spec/spec_helper.rb

```diff
@@ -90,10 +92,19 @@ def app
   end
 
   config.around :each, celluloid: true do |ex|
+    # horrible hack to keep the same pubsub actor across all specs
+    pubsub = Celluloid::Actor[:mongo_pubsub]
     Celluloid.boot
+    Celluloid::Actor[:mongo_pubsub] = pubsub
```

Comment:
Turns out that we call `Celluloid.boot` twice in the specs, and re-calling `Celluloid.boot` will clear the `Celluloid::Actor` registry, breaking the new implementation of `MongoPubsub.actor` - hence this hack to maintain compatibility with the way the specs used to all run with the same `MongoPubsub` actor, without having to actually fix the `initialize` -> `async.tail!` race....

```
call Celluloid.boot @ /home/kontena/kontena/kontena/server/app/boot.rb:25:in `require'
I, [2017-11-15T18:27:47.123222 #18779]  INFO -- MongoPubsub: initialized
pre-boot Celluloid::Actor[:mongo_pubsub] = #<MongoPubsub:0x0000000364fb88>
call Celluloid.boot @ /home/kontena/kontena/kontena/server/spec/spec_helper.rb:89:in `block (2 levels) in <top (required)>'
I, [2017-11-15T18:27:47.127439 #18779]  INFO -- MongoPubsub: starting to tail collection
post-boot Celluloid::Actor[:mongo_pubsub] = 
```